src/components/DotNavigation/DotNavigation.st.css

```diff
@@ -0,0 +1,130 @@
+:import {
+  -st-from: '../../common/formatters.st';
+  -st-named: color, fallback, value;
+}
+
+:vars {
+  /*Defaults*/
+  DefaultSelectedDotColor: color-5;
+  DefaultSelectedBorderColor: color-1;
+  DefaultSelectedDotColorLight: color-1;
+  DefaultSelectedBorderColorLight: color-5;
+}
+
+:vars {
+  /*Overrides*/
+  DotNavigationSelectedDotColor: --overridable;
+  DotNavigationSelectedBorderColor: --overridable;
+}
```

Comment:
These two variables are sort of the components style "API".
Consumers will use these variables in order to override the colors.
So let's have a bit shorter names.
I think we can use `SelectedDotColor` and `SelectedBorderColor`.

Reply:
Done)